code/modules/mob/living/simple_animal/hostile/retaliate/beast.dm

```diff
@@ -0,0 +1,76 @@
+/mob/living/simple_animal/hostile/retaliate/beast
+	name = "Wolf-like beast"
+	desc = "Some sinister monstrosity of a primal, feral nature. It's piercing gaze is enough to send shivers down your spine."
+	icon = 'icons/vampires/beastform.dmi'
+	icon_state = "beast"
+	icon_living = "beast"
+	icon_dead = "beast_dead"
+	icon_gib = "beast_dead"
+	speak = list("RAWR!","Rawr!","GRR!","Growl!")
+	speak_emote = list("growls", "roars")
+	speak_language = /datum/language/metalanguage
+	emote_hear = list("rawrs.","grumbles.","grawls.")
+	emote_taunt = list("stares ferociously", "stomps")
+	speak_chance = 1
+	taunt_chance = 25
+	turns_per_move = 1
+	see_in_dark = 10
+	butcher_results = list(/obj/item/food/meat/slab = 5)
+	response_help_continuous = "pets"
+	response_help_simple = "pet"
+	response_disarm_continuous = "gently pushes aside"
+	response_disarm_simple = "gently push aside"
+	armour_penetration = 60
+	mob_biotypes = MOB_ORGANIC | MOB_BEAST
+	maxHealth = 200
+	health = 200
+	spacewalk = FALSE
+	melee_damage = 30
+	attack_verb_continuous = "slashes"
+	attack_verb_simple = "slash"
+	faction = list(FACTION_HOSTILE)
+	attack_sound = 'sound/weapons/bladeslice.ogg'
+	obj_damage = 60
+	environment_smash = ENVIRONMENT_SMASH_WALLS
+	mob_size = MOB_SIZE_LARGE
+	footstep_type = FOOTSTEP_MOB_CLAW
+	speed = -1
```

Comment:
What does speed -1 mean?

Reply:
This means it moves about as fast (not quite) as running crew in good health

When I want to make a custom simplemob as a threat that crew can't casually walk away from this is the value I use.